cpp/oneapi/dal/backend/primitives/sort/sort_dpc.cpp

```diff
@@ -546,4 +616,42 @@ INSTANTIATE_SORT(std::int32_t)
 INSTANTIATE_SORT(std::uint32_t)
 INSTANTIATE_SORT(std::int64_t)
 INSTANTIATE_SORT(std::uint64_t)
+
+#define INSTANTIATE_SORT_INDICES_DPL(Float, Index)                                   \
+    template ONEDAL_EXPORT sycl::event radix_sort_indices_inplace_dpl<Float, Index>( \
+        sycl::queue&,                                                                \
+        ndview<Float, 1>&,                                                           \
+        ndview<Index, 1>&,                                                           \
+        const event_vector&);
+
+#define INSTANTIATE_FLOAT(Index)               \
+    INSTANTIATE_SORT_INDICES_DPL(float, Index) \
+    INSTANTIATE_SORT_INDICES_DPL(double, Index)
+
+INSTANTIATE_FLOAT(std::uint32_t)
+INSTANTIATE_FLOAT(std::int32_t)
+
+#define INSTANTIATE_RADIX_SORT(Index)                                           \
+    template ONEDAL_EXPORT sycl::event radix_sort_dpl<Index>(sycl::queue&,      \
+                                                             ndview<Index, 2>&, \
+                                                             ndview<Index, 2>&, \
+                                                             const event_vector&);
+
+INSTANTIATE_RADIX_SORT(std::int32_t)
+INSTANTIATE_RADIX_SORT(std::uint32_t)
+INSTANTIATE_RADIX_SORT(std::int64_t)
+INSTANTIATE_RADIX_SORT(std::uint64_t)
```

Comment:
Do we really need all those instantiations?

Reply:
We have these types for non dpl(for no pvc devices) radix sort, so its aligned